docs/build-and-publish-pipeline.md

```diff
@@ -56,7 +58,8 @@ and the integrity-protection mechanisms that apply at each step:
     * TODO: talk about how TC's "chain of trust" might be useful here.
 6. Bintray mirrors the built artifacts to [jcenter](https://bintray.com/bintray/jcenter).
     * TODO: as above, we're in the process of [moving this to maven.mozilla.org](https://github.com/mozilla/application-services/issues/252).
-6. Consumers fetch the published artifacts from bintray.
+7. On request, our operations team [manually mirrors artifacts to maven.mozilla.org](https://bugzilla.mozilla.org/show_bug.cgi?id=1540775).
```

Comment:
@rfk You might want to capture that inputs to the system include the release version defined at https://searchfox.org/mozilla-mobile/source/application-services/.buildconfig-android.yml#1 (e.g., `0.24.0`) and that you find the artifacts using the API GET at https://bugzilla.mozilla.org/show_bug.cgi?id=1542859#c3.